boards/DCM/Src/App/states/App_InitState.c

```diff
@@ -15,20 +16,66 @@ static void InitStateRunOnEntry(struct StateMachine *const state_machine)
 
 static void InitStateRunOnTick1Hz(struct StateMachine *const state_machine)
 {
-    App_AllStatesRunOnTick1Hz(state_machine);
+    App_SharedStatesRunOnTick1Hz(state_machine);
 }
 
 static void InitStateRunOnTick100Hz(struct StateMachine *const state_machine)
 {
-    App_AllStatesRunOnTick100Hz(state_machine);
+    App_SharedStatesRunOnTick100Hz(state_machine);
 
     struct DcmWorld *world = App_SharedStateMachine_GetWorld(state_machine);
     struct DcmCanTxInterface *can_tx_interface = App_DcmWorld_GetCanTx(world);
+    struct DcmCanRxInterface *can_rx_interface = App_DcmWorld_GetCanRx(world);
+    struct ErrorTable *       error_table = App_DcmWorld_GetErrorTable(world);
+    struct InverterSwitches * inverter_switches =
+        App_DcmWorld_GetInverterSwitches(world);
 
     App_CanTx_SetPeriodicSignal_TORQUE_REQUEST(can_tx_interface, 0.0f);
 
-    // No need for any safety checks, just run! (this is a demo)
+#ifdef DEBUG // Enter drive state directly from init state
     App_SharedStateMachine_SetNextState(state_machine, App_GetDriveState());
+    return;
+#endif
+
+    // Provide LV to inverters when both AIRS are closed and DC bus voltage ~
+    // 400 V
+    if (App_SharedStates_AreBothAIRsClosed(can_rx_interface) &&
+        !App_InverterSwitches_IsRightOn(inverter_switches))
+    {
+        App_InverterSwitches_TurnOnRight(inverter_switches);
+    }
+    if (App_SharedStates_AreBothAIRsClosed(can_rx_interface) &&
+        !App_InverterSwitches_IsLeftOn(inverter_switches))
+    {
+        App_InverterSwitches_TurnOnLeft(inverter_switches);
+    }
+
+    // Holds previous start switch position (true = UP/ON, false = DOWN/OFF)
+    static bool prev_start_switch_position =
+        true; // Initialize to true to prevent a false start
```

Comment:
nit; can we move this comment to line 53, i think the indention looks kinda funny

Reply:
done